Formula/inetutils.rb

```diff
@@ -49,6 +49,22 @@ def install
     end
   end
 
+  def caveats; <<~EOS
+    All commands have been installed with the prefix 'g'.
```

Comment:
Not all commands, see `Binaries not shadowing macOS utils symlinked without 'g' prefix`

Reply:
Is it desirable to export `noshadow` as a global variable and print it in `caveats`?